docs/getting-started/app-custom-theme.md

```diff
@@ -253,72 +253,3 @@ const LogoFull = () => {
   return <img src={MyCustomLogoFull} />;
 };
 ```
-
-## Custom Homepage
```

Comment:
I wonder if its worth keeping this heading and link to the separate page? As I'm wondering if people will miss out on the home page docs when setting up the app if its not findable here? Or at least that was our thinking when putting the docs here in the first place. What do you think?

Reply:
Sure, we should keep a line here referencing to customizing homepage. Adding.